client-test/src/test/java/net/spals/oembed4j/client/JerseyOEmbedClientFTest.java

```diff
@@ -41,107 +43,48 @@ void classTearDown() {
     @DataProvider
     Object[][] executeProvider() {
         // Spot check common oEmbed providers
-        return new Object[][] {
-                flickr(),
-                vimeo(),
-                youTube(),
+        return new Object[][]{
+            flickr(),
+            vimeo(),
+            youtube(),
         };
     }
 
     @Test(dataProvider = "executeProvider")
-    public void testExecute(final URI resourceURI,
-                            final OEmbedResponse expectedResponse) {
+    public void testExecute(final URI resourceURI) {
         final OEmbedRequest request = new OEmbedRequest.Builder().setResourceURI(resourceURI).build();
         final Optional<OEmbedResponse> response = oEmbedClient.execute(request);
         assertThat(response, not(Optional.empty()));
-        assertThat(response.get(), is(expectedResponse));
+
+        GoldFile.builder()
+            .withPath(GoldPaths.simpleClass(GoldPaths.MAVEN, JerseyOEmbedClientFTest.class))
+            .withReflectionSerializer()
+            .withSerializer(Serializers.of())
```

Comment:
I filed in issue in midas, ReflectionSerializer doesn't get a default serializer by default. So things like enums and other non java primitives fail here.